source/integrate-with-integration-environment/integrate-with-code-flow.html.md.erb

```diff
@@ -479,6 +480,10 @@ You can use the following table to understand the ID token’s claims.
     <td class="tg-0pky"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent">iat</span></td>
     <td class="tg-0pky"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent"><code>iat</code> stands for ‘issued at’. This identifies the time at which GOV.UK One Login created the JWT. You can use this claim to understand the age of the JWT.</span><br><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent">This will appear as an integer timestamp representing the number of seconds since the <a href="https://www.epochconverter.com/"><span>Unix Epoch</span></a>.</span></td>
```

Comment:
I think the line break before the last sentence in unnecessary, to keep it consistent with other parameter descriptions on the page.